cluster/addons/fluentd-elasticsearch/es-image/run.sh

```diff
@@ -1,30 +1,11 @@
-#!/bin/sh
-
-# Copyright 2017 The Kubernetes Authors.
-#
-# Licensed under the Apache License, Version 2.0 (the "License");
-# you may not use this file except in compliance with the License.
-# You may obtain a copy of the License at
-#
-#     http://www.apache.org/licenses/LICENSE-2.0
-#
-# Unless required by applicable law or agreed to in writing, software
-# distributed under the License is distributed on an "AS IS" BASIS,
-# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-# See the License for the specific language governing permissions and
-# limitations under the License.
-
-sysctl -w vm.max_map_count=262144
-
+#!/bin/bash
 export NODE_NAME=${NODE_NAME:-${HOSTNAME}}
 export NODE_MASTER=${NODE_MASTER:-true}
 export NODE_DATA=${NODE_DATA:-true}
 export HTTP_PORT=${HTTP_PORT:-9200}
 export TRANSPORT_PORT=${TRANSPORT_PORT:-9300}
 export MINIMUM_MASTER_NODES=${MINIMUM_MASTER_NODES:-2}
 
-/elasticsearch_logging_discovery >> /elasticsearch/config/elasticsearch.yml
-
-chown -R elasticsearch:elasticsearch /data
-
-exec gosu elasticsearch sh /elasticsearch/bin/elasticsearch
+sysctl -w vm.max_map_count=262144
```

Comment:
I'm not sure if its required for the es5 image, it may already be set.  however, ES5 has a strictly enforced minimum number of memory maps that must be available or ES refuses to start.